server/sonar-web/src/main/js/apps/issues/components/MyIssuesFilter.js

```diff
@@ -21,15 +21,17 @@
 import React from 'react';
 import { translate } from '../../../helpers/l10n';
 
+/*::
 type Props = {|
   myIssues: boolean,
   onMyIssuesChange: boolean => void
 |};
+*/
 
 export default class MyIssuesFilter extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  handleClick = (myIssues: boolean) => (e: Event & { currentTarget: HTMLElement }) => {
+  handleClick = (myIssues /*: boolean */) => (e /*: Event & { currentTarget: HTMLElement } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)